src/runtime_src/core/pcie/driver/linux/xocl/subdev/xmc.c

```diff
@@ -4495,7 +4495,7 @@ static void clock_status_check(struct platform_device *pdev, bool *latched)
 		if (status & 0x1) {
```

Comment:
Not for this CR, but in the future we should avoid dup code in xmc.c which should be done in clock.c. My future refactoring work will count this in.